src/main/java/com/google/sps/templates/calendar.soy

```diff
@@ -27,20 +30,84 @@
               <span class="mdc-list-item__text"><h2>Event Calendar</h2></span>
             </a>
 
-            <a class="mdc-list-item" href="#">
-              <label class="mdc-text-field mdc-text-field--filled">
-                <span class="mdc-text-field__ripple"></span>
-                <input class="mdc-text-field__input" id="address" type="text" aria-labelledby="my-label-id">
-                <span class="mdc-line-ripple"></span>
-              </label>  
+            <hr class="mdc-list-divider">
+            <div class="nav-icons">
+              <a class="icon-links" href="/chat">
+                <i class="material-icons mdc-list-item__graphic nav-icon">message</i>
+              </a>
+              <a class="icon-links" href="/calendar">
+                <i class="material-icons mdc-list-item__graphic nav-icon">date_range</i>
+              </a>
+              <a class="icon-links" href="/groups">
+                <i class="material-icons mdc-list-item__graphic nav-icon">group</i>
+              </a>
+            </div>
+
+            <li role="separator" class="mdc-list-divider"></li>
+            <form action="/events" method="POST" id="create-event">
+              <input type="hidden" name="groupId" value="{$groupId}">
+              <h6 class="mdc-list-group__subheader">Title</h6>
+
+              <a class="mdc-list-item">
+                <label class="mdc-text-field mdc-text-field--filled">
+                  <span class="mdc-text-field__ripple"></span>
+                  <input class="mdc-text-field__input" id="title" type="text" name="title" aria-labelledby="my-label-id" required minlength="2">
```

Comment:
tip: if you want to make sure the user fills out these fields before submitting just add `required` before the closing tag https://www.w3schools.com/tags/att_input_required.asp